src/main/kotlin/org/team5419/fault/util/Source.kt

```diff
@@ -0,0 +1,86 @@
+@file:Suppress("TooManyFunctions")
+package org.team5419.fault.util
+
+import kotlin.math.absoluteValue
+import kotlin.math.sign
+
+typealias Source<T> = () -> T
```

Comment:
What is this, the name doesn't say anything. This file need a bare minimum of docs.